src/components/dashboard/GapByRace.vue

```diff
@@ -1,26 +1,74 @@
 <template>
-  <div class="gap-container">
-    <div>
-      <GapChart
-        :active-stats="activeStats"
-        :expected="expected"
-        :field-data="fieldData"
-      />
+  <div class="outer-container">
+    <div
+      class="gap-container"
+      :class="{ visible: focusStat.value === 'total' }"
+    >
+      <div>
+        <GapChart
+          :active-stats="activeStats"
+          :expected="expected"
+          :field-data="fieldData"
+        />
+      </div>
+      <div class="centered">
+        <p class="has-text-centered">
+          In {{ activeCluster.name }}, the largest gap is among
+          {{ minVaxRace?.name }} residents. Only
+          <strong
+            >{{ formatPct(minVaxRace?.pct) }} of
+            {{ minVaxRace?.name }} residents</strong
+          >
+          are vaccinated compared to {{ formatPct(expected) }} statewide.
+          <strong
+            >{{ minVaxRace?.gap }} more {{ minVaxRace?.name }} residents</strong
+          >
+          need to be vaccinated to close this gap.
+        </p>
+      </div>
     </div>
-    <div class="centered">
-      <p class="has-text-centered">
-        In {{ activeCluster.name }}, the largest gap is among
-        {{ minVaxRace?.name }} residents. Only
-        <strong
-          >{{ formatPct(minVaxRace?.pct) }} of
-          {{ minVaxRace?.name }} residents</strong
-        >
-        are vaccinated compared to {{ formatPct(expected) }} statewide.
-        <strong
-          >{{ minVaxRace?.gap }} more {{ minVaxRace?.name }} residents</strong
-        >
-        need to be vaccinated to close this gap.
-      </p>
+    <div
+      class="gap-container"
+      :class="{
+        visible:
+          focusStat.value !== 'total' && activeFocusStats?.population > 0,
+      }"
+    >
+      <div
+        class="is-flex is-flex-direction-row is-justify-content-space-around"
+      >
+        <KeyPerformanceIndicator
+          :value="formatPct(activeFocusStats?.pct)"
+          title="Vaccinated in Community"
```

Comment:
should this title more clearly specify the race selected? @s-bessey

Reply:
I've updated this wording !